src/htletterspacer/__main__.py

```diff
@@ -58,18 +65,25 @@ def main(args: list[str] | None = None) -> None:
     return None
 
 
-def space_ufo(args: argparse.Namespace) -> None:
-    ufo: ufoLib2.Font = args.ufo
+def space_ufo(
+    ufo: ufoLib2.Font,
+    area: int | None = None,
+    depth: int | None = None,
+    overshoot: int | None = None,
+    config_file: Path | None = None,
```

Comment:
For pedantry, you may want to make this `os.PathLike[str]` instead and `Path(...)` it inside the function.

Reply:
Nice, now done so